app/controllers/leads_controller.rb

```diff
@@ -143,6 +144,14 @@ def lead_params
     params.require(:lead).permit(:first_name, :last_name, :email, :phone, :city, :state, :zip, :contacted, :appointment_date, :notes, :connected, :bad_number, :advisor, :location, :first_appointment_set, :first_appointment_actual, :first_appointment_format, :second_appointment_set, :second_appointment_actual, :second_appointment_format, :enrolled_date, :deposit_date, :sales, :collected, :status, :next_step, :rep_notes, :exclude_from_calling, :meeting_type, :meeting_format)
   end
 
+  def has_auto_text?
+    if current_admin.setting
+      return current_admin.setting.auto_text
+    else
+      return "This is Rena from the Actualize coding bootcamp. Do you have a minute to talk?"
```

Comment:
Good job! Only thing is we don't need the return in the if statement because ruby implicitly returns the last line.